### ISSUE TITLE
EmuWebApp annotation via URL: Change file name for export

### ISSUE THREAD

**Issue by julianpoemp** [IPS-LMU/EMU-webApp#249]
## Scenario:
1. User opens the EmuWebApp with GET parameters for audio source and transcript file
2. After annotation the user would like to save the changed annotation and clicks on one of the download buttons
3. The default file name for the download to AnnotJSON in a URL session is "from URL parameters_annot.json"

## Suggestion:
It would be nice if  the file name would be named with the same name as the audio file (with the correct extension of course)

**Comment by raphywink** [comment]
good point. Shouldn't bee to difficult to implement either... might still take some time till I get around to implementing this though...

**Comment by samgregory** [comment]
Hi @MJochim - another simple fix here. 

During the load from URL parameters the curBndlName is set to the basename (- extension) of the audio file in this line:
https://github.com/IPS-LMU/EMU-webApp/blob/59d19c7f653d9a5ae6081f6f0ca9da0697d24d47/src/app/components/emu-webapp.component.ts#L643

If we change line 684 from:
https://github.com/IPS-LMU/EMU-webApp/blob/59d19c7f653d9a5ae6081f6f0ca9da0697d24d47/src/app/components/emu-webapp.component.ts#L684
to 
var bndlList = [{'session': 'File(s)', 'name': this.LoadedMetaDataService.getCurBndlName()}];

then you will get the desired **audiofilename_annot.json** format in the download modal.

I hope it's ok that I'm not bothering with pull requests for these one line fixes?
